test/unit/index.ts

```diff
@@ -93,6 +93,31 @@ describe('oauth tooling', () => {
 
       expect(result).to.equal(expected);
     });
+
+    it('should return the correct uri as string with queryParams specified', () => {
+
+      // given
+      const authorizationEndpoint = '/oauth2/authorize';
+      const clientId = 'clientID';
+      const redirectUri = '/redirect';
+      const queryParams = {
+        foo: 'bar'
+      };
+
+      // when
+      const result = createAuthCodeRequestUri(authorizationEndpoint, clientId,
+        redirectUri, queryParams);
+
+      // then
+      const expected = authorizationEndpoint +
```

Comment:
Yes, I would suggest this solution:
```typescript
      const expected = `${authorizationEndpoint}` +
        `?client_id=${clientId}` +
        `&redirect_uri=${redirectUri}` +
        `&response_type=code` +
        `&realm=${EMPLOYEES_REALM}` +
        `&foo=bar`;
```

cause with multi line template strings we have the problem that 
 whitespaces are not ignored. So that would destroy the formatting, right?